mex/backend/auxiliary/wikidata.py

```diff
@@ -65,5 +68,12 @@ def extracted_primary_source_wikidata() -> ExtractedPrimarySource:
         extracted_primary_sources,
         "wikidata",
     )
-
+    connector = BackendApiConnector.get()
```

Comment:
The backend should not depend on it self. Use the GraphConnector instead. This will also raise a different error if it fails. Not sure if you need to catch that, though.
```suggestion
    connector = GraphConnector.get()
```